src/assets/pack-templates/rune-descriptions/rune-element-descriptions.yaml

```
@@ -1,171 +1,77 @@
-  Earth is the third Element, Daughter of the Seas, Mother of the Sky, Wife to Many, Supporter of All. At
-  least half the cults in Glorantha include the Earth in some manifestation or other, as Life-Giver, Stone-
-  Father, Shaker, or Primal Ancestress.</p>
-  <p>The element of matter and the terrestrial
-  world, Earth is associated with the sensual
-  aspects of both life and death. It is rich with
-  life and gifts, sometimes terrifying and deadly.
-  It is the Rune of physical things, and all things that live and
-  grow atop and within the earth.</p>
-  <p><b>Characteristic:</b> CON</p>
-  <p><b>Personality:</b> To have a strong affinity with Earth is to be pragmatic, practical, prudent, worldly, sturdy, and sensual.</p>
-  <p><b>Sense:</b> Search</p>
-  <p><b>Skill Category:</b> Communication</p>
-  <p><b>Weapon:</b> Axe</p>
-  <p><b>Organs:</b> Genitals, bone</p>
-  <p><b>Color:</b> Green</p>
-  <p><b>Metal:</b> Copper</p>
-  <p><b>Phyla:</b> Reptile</p>
+  ${{openSystem.content.journalEntryType.runeDescription.title}}$
+  <p><img style="float: right; margin-left:1rem; border:none;" src="systems/rqg/assets/images/runes/earth.svg" height="175"></p>${{openSystem.content.runeDescription.earthElement.description}}$
 flags:
   rqg:
     documentRqidFlags:
       id: je..earth-element
       lang: ${{openSystem.lang}}$
       priority: ${{openSystem.prio}}$
 ---
-name: Darkness (element)
-img: systems/rqg/assets/images/runes/darkness.svg
+# firesky
+name: ${{openSystem.content.runeDescription.fireskyElement.name}}$
+img: systems/rqg/assets/images/runes/fire_sky.svg
 content: >
-  <p><img style="float: right; margin-left:1rem; border:none;" src="systems/rqg/assets/images/runes/darkness.svg" height="175">
-  The most ancient Rune, the First Born, from whom all other Elements were born or descended.
-  Darkness was the first Element to arise out of the Primal Chaos.</p>
-  <p>The element of Darkness was the first to withdraw itself
-  from Chaos, its birth attended by tales of
-  primeval wars of Darkness against Chaos.
-  It is the element of the Underworld, the
-  endless freezing darkness that preceded
-  creation and still lurks and hungers under
-  the earth and seas.</p>
-  <p><b>Characteristic:</b> SIZ</p>
-  <p><b>Personality:</b> To have a strong affinity with Darkness is to be cold, cruel, patient, hungry, large, and secretive.</p>
-  <p><b>Sense:</b> Listen</p>
-  <p><b>Skill Category:</b> Stealth</p>
-  <p><b>Weapons:</b> Club, mace, rock</p>
-  <p><b>Organs:</b> Stomach, fat</p>
-  <p><b>Color:</b> Black</p>
-  <p><b>Metal:</b> Lead</p>
-  <p><b>Phyla:</b> Insect</p>
+  ${{openSystem.content.journalEntryType.runeDescription.title}}$
+  <p><img style="float: right; margin-left:1rem; border:none;" src="systems/rqg/assets/images/runes/fire_sky.svg" height="175"></p>${{openSystem.content.runeDescription.fireskyElement.description}}$
 flags:
   rqg:
     documentRqidFlags:
-      id: je..darkness-element
+      id: je..firesky-element
```

the `/` in `fire/sky` should become `-`like this
```suggestion
      id: je..fire-sky-element
```

---

ok. I will change in the rune and the rune description as the rune has firesky as well